t/02_yaml_schedule.t

```diff
@@ -87,4 +87,44 @@ subtest 'parse_yaml_test_schedule_recursive_conditional' => sub {
     ok $modules[3] eq 'bar/test3', "Basic scheduling";
 };
 
+subtest 'merge default schedules with flows and individual schedules' => sub {
+    use scheduler;
+    use testapi 'set_var';
+
+    # positive unit tests
+
+    my $schedule = $ypp->load_file(dirname(__FILE__) . '/data/test_data_mini_default_flows.yaml');
+    my @modules;
+    set_var('YAML_SCHEDULE_DEFAULT', '/t/data/flows/default.yaml');
+    set_var('YAML_SCHEDULE_FLOWS', 'flow_a');
+    @modules = scheduler::parse_schedule($schedule);
+    ok $modules[0] eq 'flow_a_foo', "Merge default with flows and individual";
```

Comment:
this framework has options to compare arrays, we don't need to go one by one :)
https://perldoc.perl.org/Test::More#eq_array